### PR TITLE
filters added

### PR DESCRIPTION
**Pull request by BINAH25**
filters added
image

### CODE REVIEW

File: web/src/widgets/TreatmentsWidget/index.jsx [hunk 1]
```diff
@@ -22,7 +37,10 @@ function TreatmentsWidget() {
                             <TableView
                                 responseDataAttribute="treatments"
                                 dataSourceUrl={`${BASE_API_URI}/treatments/`}
-                                filterByDate={true}
```
Comment (dodziraynard) [inline web/src/widgets/TreatmentsWidget/index.jsx:25]: Why did you remove this?

Comment (BINAH25) [reply web/src/widgets/TreatmentsWidget/index.jsx:25]: by mistake
added it

File: web/src/widgets/ReferralsWidget/index.jsx [hunk 2]
```diff
@@ -195,6 +194,12 @@ function ReferralsWidget() {
                                     { key: `status:review`, value: `Review` },
                                     { key: `status:completed`, value: `Completed` },
                                 ]}
+                                filters2={facilities?.map(facility => ({
+                                    key: `facility:${facility.id}`,
+                                    value: facility.name
+                                })) || []} 
+                               
+
```
Comment (dodziraynard) [inline web/src/widgets/ReferralsWidget/index.jsx:202]:
```suggestion
```

File: web/src/widgets/ReferralsWidget/index.jsx [hunk 3]
```diff
@@ -274,4 +279,4 @@ function ReferralsWidget() {
     );
 }
 
-export default ReferralsWidget;
+export default ReferralsWidget;
```
Comment (dodziraynard) [inline web/src/widgets/ReferralsWidget/index.jsx:282]:
```suggestion
export default ReferralsWidget;

```

File: web/src/widgets/TreatmentsWidget/index.jsx [hunk 4]
```diff
@@ -1,14 +1,29 @@
...
 
 const TableView = React.lazy(() => import("../../components/Table"));
 
 function TreatmentsWidget() {
+    
+    const [facilities, setFacilities] = useState([])
+    const [getFacilities, { data: facilitiesResponse = [], isLoading: isLoadingFacilities, error: errorLoadingFacilities }] = useLazyGetAllFacilitiesQuery()
+    useEffect(() => {
+        getFacilities()
+
```
Comment (dodziraynard) [inline web/src/widgets/TreatmentsWidget/index.jsx:21]:
```suggestion
```

File: web/src/widgets/TreatmentsWidget/index.jsx [hunk 5]
```diff
@@ -1,14 +1,29 @@
...
+    const [facilities, setFacilities] = useState([])
+    const [getFacilities, { data: facilitiesResponse = [], isLoading: isLoadingFacilities, error: errorLoadingFacilities }] = useLazyGetAllFacilitiesQuery()
+    useEffect(() => {
+        getFacilities()
+        
+    }, [])
+
+    monitorAndLoadResponse(facilitiesResponse, "facilities", setFacilities)
+
+
```
Comment (dodziraynard) [inline web/src/widgets/TreatmentsWidget/index.jsx:26]:
```suggestion
```

File: web/src/widgets/TreatmentsWidget/index.jsx [hunk 6]
```diff
@@ -87,4 +106,4 @@ function TreatmentsWidget() {
     );
 }
 
-export default TreatmentsWidget;
+export default TreatmentsWidget;
```
Comment (dodziraynard) [inline web/src/widgets/TreatmentsWidget/index.jsx:109]:
```suggestion
export default TreatmentsWidget;

```
